### DISCUSSION TITLE
동학개미운동 by AnTalk

### DISCUSSION THREAD

**Discussion by save-us-prophet** [ShareAnTalk/Kiwoom#46] [General]
### TODO
1. 주식
* [ ] 호가창 좌측하단 체결내역
* [ ] 지수차트 실시간 차트 구현
* [x] 채팅 날짜변경선 일자만 조회하는 듯. .
* [ ] 하단 주가지수 장중 마지막 데이터로 반영. .

2. 선물옵션
* [ ] 호가창 좌측하단 체결내역

3. 자산
* [x] 선물옵션 보유종목

4. 프로필
* [ ] 히스토리 리플래쉬

**Comment by cyberprophet** [comment]
Good idea
